ios/FluentUI.Demo/FluentUI.Demo/Demos/ListDemoController.swift

```diff
@@ -104,6 +105,9 @@ class ListDemoController: DemoController {
                 listCell.trailingView = section.hasAccessory ? createCustomView(imageName: cell.image) : nil
                 listCell.accessoryType = accessoryType(for: rowIndex)
                 listCell.layoutType = updateLayout(subtitle: listCell.subtitle)
+                if rowIndex != TableViewCellSampleData.numberOfItemsInSection - 1 {
```

Comment:
This logic could be handled inside the component.
It seems weird that the client code needs to remove the divider for the last item.